.eslintrc.json

```diff
@@ -0,0 +1,108 @@
+            "files": ["*.json"],
+            "parser": "jsonc-eslint-parser",
+            "rules": {
+                "jsonc/sort-keys": ["warn"]
+            }
+        },
+        {
+            "extends": ["plugin:jsonc/recommended-with-jsonc"],
+            "files": ["package.json"],
+            "parser": "jsonc-eslint-parser",
+            "rules": {
+                "jsonc/sort-keys": [
+                    "warn",
+                    {
+                        "hasProperties": ["type"],
+                        "order": [
+                            "$schema",
+                            "name",
+                            "version",
+                            "private",
+                            "description",
+                            "license",
+                            "author",
+                            "maintainers",
+                            "contributors",
+                            "homepage",
+                            "repository",
+                            "bugs",
+                            "type",
+                            "exports",
+                            "main",
+                            "module",
+                            "browser",
+                            "man",
+                            "preferGlobal",
+                            "bin",
+                            "files",
+                            "directories",
+                            "scripts",
+                            "config",
+                            "sideEffects",
+                            "types",
+                            "typings",
+                            "workspaces",
+                            "resolutions",
+                            "dependencies",
+                            "bundleDependencies",
+                            "bundledDependencies",
+                            "peerDependencies",
+                            "peerDependenciesMeta",
+                            "optionalDependencies",
+                            "devDependencies",
+                            "keywords",
+                            "engines",
+                            "engineStrict",
+                            "os",
+                            "cpu",
+                            "publishConfig"
+                        ],
+                        "pathPattern": "^$"
+                    },
+                    {
+                        "order": { "type": "asc" },
+                        "pathPattern": "^(?!exports\\[).*"
+                    }
+                ]
+            }
+        },
+        {
+            "extends": ["./first-gen/.eslintrc.json"],
+            "files": ["first-gen/**/*"]
+        },
+        {
+            "extends": ["./second-gen/.eslintrc.json"],
+            "files": ["second-gen/**/*"]
+        }
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
```

Comment:
Can we just use "latest" here or is there a reason we need to use "2020" version?